config/puma.rb

```diff
@@ -25,7 +25,7 @@
 # Workers do not work on JRuby or Windows (both of which do not support
 # processes).
 #
-workers ENV.fetch("WEB_CONCURRENCY", 2) unless environment_name == "development"
+workers ENV.fetch("WEB_CONCURRENCY", 2) unless environment_name.in?(%w[test development])
```

Comment:
I figured this wouldn't disrupt anyone's environment. Debuggers don't like concurrency.

Reply:
Good catch!